projects/storefrontlib/shared/components/split-view/view/view.component.ts

```diff
@@ -129,6 +152,24 @@ export class ViewComponent implements OnInit, OnDestroy {
     }
   }
 
+  protected showAssistiveMessage() {
```

Comment:
Since this creates a subscription that does something on some action, lets include this in the title to be more descriptive. Something like `onViewChangeShowAssistiveMessage`.

Reply:
name updated